### DISCUSSION TITLE
How can i REPLACE stringOptions in q-select autocomplete by arrayOptions?

### DISCUSSION THREAD

**Discussion by zoculos** [quasarframework/quasar#10171] [CLI - PWA mode] [answered]
Greetingsl! According to quasar doc  

<div id="q-app">
  <div class="q-pa-md">
      <div class="q-gutter-md row">
        <q-select
          filled
          v-model="model"
          use-input
          hide-selected
          fill-input
          input-debounce="0"
          :options="options"
          @filter="filterFn"
          style="width: 250px; padding-bottom: 32px"
        >
          <template v-slot:no-option>
            <q-item>
              <q-item-section class="text-grey">
               Not found !
              </q-item-section>
            </q-item>
          </template>
        </q-select>
      </div>
    </div>
</div>

const stringOptions = [
  'Google', 'Facebook', 'Twitter', 'Apple', 'Oracle'
]

new Vue({
  el: '#q-app',
  data () {
    return {
      model: null,
      options: stringOptions
    }
  },

  methods: {
    filterFn (val, update, abort) {
      if (val.length < 1) {
        abort()
        return
      }

      update(() => {
        const needle = val.toLowerCase()
        this.options = stringOptions.filter(v => v.toLowerCase().indexOf(needle) > -1)
      })
    }
  }
})
**HOW CAN I REPLACE** const stringOptions = [
  'Google', 'Facebook', 'Twitter', 'Apple', 'Oracle'
] **BY ARRAY** stringOptions = [ { id:1,name: 'Google' }, 
  { id:2,name: 'Facebook'}, { id:3, name: 'Twitter'}, 
  { id:4, name: 'Apple'},                             
  { id:5, name:  'Oracle'}]. Thanks...

**Comment by MilosPaunovic** [comment, marked as answer]
Check [this codepen](https://codepen.io/pen/?&editors=101&prefill_data_id=ea30cc6e-1889-4bc3-9b13-1c4ef32dd1c5), from [offical docs](https://quasar.dev/vue-components/select#example--object-options), which is an example of how to use array of objects as options for `QSelect`.

**Reply by zoculos** [reply]
This link is empty!

**Reply by MilosPaunovic** [reply]
True, linking Codepens isn't my strong suit :)

It's a link from docs example:
https://quasar.dev/vue-components/select#example--object-options

So you can check it out there.

**Reply by zoculos** [reply]
Thanks! I will try anyway.

**Reply by zoculos** [reply]
Thanks to your help i finally got done! Iam so thankful to you.

**Reply by hawkeye64** [reply]
In order for someone else to see the CodePen you are linking to, you must save it first or it goes into oblivion.
